validator/sawtooth_validator/journal/chain.py

```diff
@@ -121,7 +119,8 @@ def __init__(self,
         self._chain_commit_state = ChainCommitState(
             self._block_cache.block_store, [])
         self._new_block = new_block
-        self._chain_head = chain_head
+        self._chain_head = None  # set during execution of the of the
```

Comment:
This comment should go before `self._chain_head`. In it's current format, the second half looks like it's referring to the next property.